index.js

```diff
@@ -1,13 +1,14 @@
 const express = require('express');
 const bodyParser = require('body-parser');
-const { bookRoutes, errorRoutes, customerRoutes } = require('./routers');
+
+const { bookRoutes, errorRoutes, userRoutes } = require('./routers');
```

Comment:
it would be better to give names to imports the same (or, at least pretty similar) as in original modules. So u have there `bookRouter`, `userRouter` etc